### PR TITLE
Allow GCE jobs to run different suites so we can test overlay

### PR DESCRIPTION
**Pull request by smarterclayton**
Should really be extended_gce.yml, but will change in a subsequent spot

### CODE REVIEW

**Comment by smarterclayton** [comment]
@stevekuznetsov

**Comment by smarterclayton** [comment]
Rebased

**Comment by smarterclayton** [comment]
Meeeerge

File: config/test_cases/test_branch_origin_extended_conformance_gce.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@ extensions:
   sync_repos:
     - name: "release"
   parameters:
+    - name: "SUITE"
```
Comment (stevekuznetsov) [inline config/test_cases/test_branch_origin_extended_conformance_gce.yml]: Yeah, this should not be in the `conformance` job ... but the tools today do not let you change that simply so ... whatever

**Pull request merged**